README.md

```diff
@@ -1811,7 +1811,7 @@ The following extractors use this feature:
 
 #### youtubetab (YouTube playlists, channels, feeds, etc.)
 * `skip`: One or more of `webpage` (skip initial webpage download), `authcheck` (allow the download of playlists requiring authentication when no initial webpage is downloaded. This may cause unwanted behavior, see [#1122](https://github.com/yt-dlp/yt-dlp/pull/1122) for more details)
-* `approximate_date`: Extract approximate `upload_date` and `timestamp` in flat-playlist. This may cause date-based filters to be slightly off
+* `approximate_metadata`: Extract approximate `upload_date`, `timestamp`, and `description` in flat-playlist. This may cause date-based filters to be slightly off and descriptions to be incomplete
```

Comment:
maybe something like:

```suggestion
* `approximate_metadata`: Extract approximate `upload_date`, `timestamp`, and `description` metadata. Note that this may cause date-based filters to be slightly off, descriptions to be incomplete and metadata to be inconsistent with full video metadata.
```

no need to mention flat-playlist imo